package.json

```
@@ -44,10 +44,10 @@
     "xdg-basedir": "^4.0.0"
   },
   "devDependencies": {
-    "@tsconfig/node10": "^1.0.8",
+    "@tsconfig/node12": "^1.0.9",
     "@types/eslint": "^7.28.0",
     "@types/minimist": "^1.2.2",
-    "@types/node": "^10.17.60",
+    "@types/node": "^16.9.1",
```

These types should match the lowest version we support I think
```suggestion
    "@types/node": "^12.20.0",
```

---

Probably even with `~` as DefinitelyTyped tracks `major.minor` against upstream

```suggestion
    "@types/node": "~12.20.0",
```